packages/lesswrong/components/editor/CKPostEditor.tsx

```diff
@@ -431,7 +432,9 @@ const CKPostEditor = ({
   const hasEverDialoguedBefore = !!anyDialogue && anyDialogue.length > 1;
 
   const [_, setEditor] = useContext(EditorContext);
-  
+
+  const [editorObject, setEditorObject] = useState<Editor | null>(null);
```

Comment:
We already have an `editorRef` using `useRef` and the `EditorContext` 2 lines above, but neither seem to do exactly what we want so we add this extra state...